daml/Marketplace/Custody/Role.daml

```diff
@@ -39,6 +39,12 @@ template Role
         do
           create Issuance.Offer with ..
 
+      nonconsuming AcceptIssuanceRequest : ContractId Issuance.Service
```

Comment:
I'm super confused - how could we have a `OfferIssuanceService` previously, but not an `Accept*`?

Reply:
The access pattern we use is the following :

1. A **Provider** can `Offer` a service where the responsibility lies with the **Customer** to either `Accept` or `Reject` said `Offer`. In most cases, this requires the signature of both the **Operator** and the **Provider** and thusly is why a `Role` contract with both signatures is required to make such an `Offer` ;
2. A **Customer** can make a `Request` for a given service where the responsibility lies with the **Provider** (and sometimes the **Operator**) to `Approve` the said `Request` 

For example, you can see this pattern by checking the required signatories and parties on controllers on the `Offer` and `Request` contracts for the `Custody` service  :

>     template Offer
>       with
>         operator : Party
>         provider : Party
>         customer : Party
>       where
>         signatory operator, provider
> 
>     controller customer can
>       Accept : ContractId Service
>         do
>           create Service with ..
> 
>       Decline : ()
>         do pure ()
> 
>     controller provider can
>       Withdraw : ()
>         do pure ()
> 
>     template Request
>       with
>         customer : Party
>         provider : Party
>       where
>         signatory customer
>         observer provider
> 
>     controller customer can
>       Cancel : ()
>         do pure ()
> 
>     controller provider can
>       Reject : ()
>         do pure ()
> 
>     choice Approve : ContractId Service
>       with
>         operator : Party
>       controller operator, provider
>       do
>         create Service with ..

This pattern is also used for role contracts where the **Operator** `Offer`s roles and correspondingly the **Provider** `Request`s roles where the responsibility lies with the relating party to `Accept` an `Offer` or `Approve` a `Request`.

Reply:
I understand the pattern, but I don't understand why you've had to add the `Accept` choice _now_, in this PR. Did not exist in `sales-eng` already? Otherwise, how could the existing `Request` be completed (line 40)?

Reply:
I added it now as I only noticed now that it was missing - the demo setup is using the `Offer` contract and thus I didn't realise it was missing.